packages/gluestick/.babelrc

```diff
@@ -10,6 +10,7 @@
   ],
   "plugins": [
     "transform-decorators-legacy",
-    "transform-flow-strip-types"
+    "transform-flow-strip-types",
+    "universal-import"
```

Comment:
This is new, although I'm not sure if .babelrc is even used (it's intentionally skipped on the client).